tests/unit/test_workflow_files.py

```diff
@@ -18,6 +18,7 @@
 import logging
 import os
 from pathlib import Path
+from posixpath import expanduser
```

Comment:
```suggestion
```

Reply:
Ah doh, vscode auto import, I need to disable that! 🙄